hw/ip/otbn/doc/_index.md

```diff
@@ -707,7 +730,7 @@ The key replacement is a two-step process:
   The request takes multiple cycles to complete.
 
 Host software can initiate a data memory secure wipe by writing 1 to the {{< regref "SEC_WIPE.dmem">}} register field.
-The [completion]({{<relref "#design-details-secure-wipe-completion">}}) is signalled by raising an {{< regref "INTR_STATE.done" >}} interrupt.
+If a secure wipe was triggered in this way, the [completion]({{<relref "#design-details-secure-wipe-completion">}}) is signaled by raising an {{< regref "INTR_STATE.done" >}} interrupt.
```

Comment:
Nit: I don't think we need the "the" here (idiomatic English is weird!)

Similarly for the next two cases.

(Also, I'm sad about "signaled": a hint of Britishness is being lost :-))